File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/PrezentaComisiiGlInIntervalReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class PrezentaComisiiGlInIntervalReportFilterModel {

	private DocumentIdentifierModel document;
	private Long institutieId;
	private String participantAcreditat;
	private String numeParticipantInlocuitor;
	private String prenumeParticipantInlocuitor;
	private String functie;

	public DocumentIdentifierModel getDocument() {
		return document;
	}

	public void setDocument(DocumentIdentifierModel document) {
		this.document = document;
	}

	public Long getInstitutieId() {
		return institutieId;
	}

	public void setInstitutieId(Long institutieId) {
		this.institutieId = institutieId;
	}

	public String getParticipantAcreditat() {
		return participantAcreditat;
	}

	public void setParticipantAcreditat(String participantAcreditat) {
		this.participantAcreditat = participantAcreditat;
	}

	public String getNumeParticipantInlocuitor() {
		return numeParticipantInlocuitor;
	}

	public void setNumeParticipantInlocuitor(String numeParticipantInlocuitor) {
		this.numeParticipantInlocuitor = numeParticipantInlocuitor;
	}

	public String getPrenumeParticipantInlocuitor() {
		return prenumeParticipantInlocuitor;
	}

	public void setPrenumeParticipantInlocuitor(String prenumeParticipantInlocuitor) {
		this.prenumeParticipantInlocuitor = prenumeParticipantInlocuitor;
	}

	public String getFunctie() {
		return functie;
	}

	public void setFunctie(String functie) {
		this.functie = functie;
	}


}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/prezenta-comisii-gl-in-interval-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";
import { DocumentIdentifierModel } from "./document-identifier.model ";

@JsonObject
export class  PrezentaComisiiGlInIntervalReportFilterModel {

    @JsonProperty("document", DocumentIdentifierModel)
    public document: DocumentIdentifierModel = null;

    @JsonProperty("institutieId", Number)
    public institutieId: number = null;

    @JsonProperty("participantAcreditat", String)
    public participantAcreditat: string = null;
       
	@JsonProperty("numeParticipantInlocuitor", String)
	public numeParticipantInlocuitor: string = null;
    
	@JsonProperty("prenumeParticipantInlocuitor", String)
	public prenumeParticipantInlocuitor: string = null;
    
	@JsonProperty("functie", String)
	public functie: string = null;

}